generators/typescript/sdk/versions.yml

```diff
@@ -1,4 +1,38 @@
 # yaml-language-server: $schema=../../../fern-versions-yml.schema.json
+- version: 3.31.0
+  changelogEntry:
+    - summary: |
+        Add `generateSubpackageExports` configuration option to enable direct imports of subpackage clients.
+        
+        When enabled, users can import and instantiate subpackage clients directly:
+        ```typescript
+        import { EnumClient } from '@org/sdk/endpoints/enum';
+        
+        const client = new EnumClient({
+            environment: "https://api.example.com",
+            token: "your-token"
+        });
+        ```
+        
+        This feature:
```

Comment:
No need to explain this much. What's more important is that you can import subpackages, and thus, JavaScript bundlers will only include the code for the subpackage you're importing, and you can potentially get a much smaller bundle.